android/app/src/test/java/com/zulipmobile/notifications/FcmMessageTest.kt

```diff
@@ -52,6 +52,19 @@ class FcmMessageTest : FcmMessageTestBase() {
     }
 }
 
+class RecipientTest : FcmMessageTestBase() {
+    @Test
+    fun `GroupPm#getPmUsersString gives the correct string`() {
+        expect.that(Recipient.GroupPm(pmUsers = setOf(123, 234, 345)).getPmUsersString())
+            .isEqualTo("123, 234, 345")
```

Comment:
Let's eliminate the spaces too.

If this already is enough that the JS code understands it, that's good :) -- but the format used elsewhere is purely comma-separated, so may as well make it consistent.